runners/core-java/src/test/java/org/apache/beam/runners/core/metrics/SpecMonitoringInfoValidatorTest.java

```diff
@@ -50,9 +50,11 @@ public void validateReturnsErrorOnInvalidMonitoringInfoType() {
   public void validateReturnsNoErrorOnValidMonitoringInfo() {
     MonitoringInfo testInput =
         MonitoringInfo.newBuilder()
-            .setUrn(Urns.USER_COUNTER_PREFIX + "someCounter")
+            .setUrn(Urns.USER_COUNTER)
+            .putLabels(MonitoringInfoConstants.Labels.NAME, "someCounter")
+            .putLabels(MonitoringInfoConstants.Labels.NAMESPACE, "")
```

Comment:
Technically, that's correct. We match only sets of labels, whether value of label is empty string or not -- that's not something specified in schema of proto. This is a problem for any field in general as of now.

I'll keep code as-is and we can review this behavior in separate PR, otherwise this one might grow too big.

Do you believe it is worth to create a jira ticket regarding this issue?